cluster/juju/layers/kubernetes-worker/reactive/kubernetes_worker.py

```diff
@@ -25,6 +26,12 @@
 def remove_installed_state():
     remove_state('kubernetes-worker.components.installed')
 
+@hook('stop')
+def remove_this_node():
+    ''' Deletes this unit's node when the unit is destroyed. '''
+    kubectl = ['kubectl', '--kubeconfig=/srv/kubernetes/config']
```

Comment:
Refactor this to use the `kubectl()` method defined on line: 336, validate that it works as intended. Same with the command on line 33. 

also - please- ensure this still passes after that modification, if we need to fix the kubeconfig method, I'm +1 to fixing it there rather than carrying a subprocess hack inline here. 

This makes it easier to update if we ever move the location of the kubeconfig. Once that's done this is +1 LGTM